internal/status/status.go

```diff
@@ -91,11 +96,79 @@ func GetStatus(w http.ResponseWriter, req *http.Request) {
 		lastUpdate = time.Now()
 	}
 
-	user, _ := auth.FromRequest(req)
-	if user != nil {
-		response["user"] = map[string]interface{}{
-			"user_id":      user.ID,
-			"assigned_sdk": auth.SDKAddress(user),
+	responses.AddJSONContentType(w)
+	w.WriteHeader(respStatus)
+	respByte, err := json.MarshalIndent(response, "", "  ")
+	if err != nil {
+		logger.Log().Error(err)
+	}
+	w.Write(respByte)
+}
+
+func GetStatusV2(w http.ResponseWriter, r *http.Request) {
+	respStatus := http.StatusOK
+	var response statusResponse
+
+	if cachedResponse != nil && lastUpdate.After(time.Now().Add(statusCacheValidity)) {
+		copier.Copy(&response, cachedResponse)
+	} else {
+		services := map[string]ServerList{
+			"lbrynet": {},
+		}
+		response = statusResponse{
+			"timestamp":     fmt.Sprintf("%v", time.Now().UTC()),
+			"services":      services,
+			"general_state": statusOK,
+		}
+		failureDetected := false
+
+		var (
+			userID     int
+			qCache     cache.QueryCache
+			sdkAddress string
+		)
+		rt := sdkrouter.New(config.GetLbrynetServers())
+		user, err := auth.FromRequest(r)
+		if err != nil {
+			sdkAddress = rt.RandomServer().Address
+		} else {
+			sdkAddress = auth.SDKAddress(user)
```

Comment:
i believe its possible that there's no error, but also there's no user. this happens if the auth token is provided but doesn't match any user in internal-apis

if you find this behavior confusing, we can have it error in that case as well. i was not sure which option is better when i wrote this so i'd love to hear your thoughts